components/badge/badge.css

```diff
@@ -0,0 +1,54 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+@import url("/components/avatar/avatar.css");
+
+* {
+    font-family: 'Montserrat', sans-serif;
+    --online: #15803d;
+    --offline: #737373;
+    --dnd: #ef4444;
+    --lightgrey: #e5e5e5;
+
+}
+
+.badge-container {
+    width: 36rem;
+    background-color: var(--lightgrey);
+    border: 0.1rem solid #a3a3a3;
+    padding: 2rem;
+    margin: 1.5rem;
+    display: flex;
+    justify-content: center;
+}
+
+.value {
+    background-color: #756ec5;
+    color: white;
+    padding: 0.2rem;
+    border-radius: 50%;
+    position: relative;
+    top: -1rem;
+    left: -0.4rem;
+    font-size: small;
+}
+
+.avatar-badge {
+    position: relative;
+    top: -1.2rem;
+    left: 3.3rem;
+    height: 0.5rem;
+    width: 0.5rem;
+    border-radius: 50%;
+}
+
+.online {
+    background-color: var(--online);
+
+}
+
+.offline {
+    background-color: var(--offline);
+}
```

Comment:
offline seems not very specific class name,  i assume this is a status so more specific class name could be used for improving readability. e.g.(status-on/online).